### DISCUSSION TITLE
Display connection

### DISCUSSION THREAD

**Discussion by hantrax** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#741] [General]
Hi Guys..

I have a ST Moritz model with pump S100101
I installed sw and when i power on ESP by USB cable, works fine and I can see the web interface
If i connect the board to the pump, ESP don't power
any idea?

**Comment by DeanozUK** [comment]
Can you post some screenshots of your PCB build?

**Reply by hantrax** [reply]
Hi...
Thanks for replay

I fixed it...there was a problem on connector

I have problem with wifi...but I think tomorrow to fix it

It's normal yhat the toner of audio.is different from original?

![20240703_214127.jpg](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/52530939/bf7b8457-dcba-40c0-8c94-eb6187d9f2a1)

**Reply by lukexperias** [reply]
Hi, I have a ST Moritz model too, with pump S100101. I have received my hardware with PCB-V2 today and I have to solder it. 

You don't use any capacitors or resistance, isn't it ? 

Thanks a lot 

![photo_2026-03-18_17-38-23](https://github.com/user-attachments/assets/b9667ab5-0eb5-4449-a585-66b70eae3d75)

**Reply by hantrax** [reply]
Hi..
No capacitor
capacitobut now works fine

**Comment by Dezikkk** [comment]
I had an issue  that I forgot to report that occurred on my St Moritz S100101. I built my own PCB firstly on breadboard following the schematic, and when I plugged the capacitors the ESP won't start. In final design I only used resistors and omitted capacitors and it is working just fine now.

**Reply by hantrax** [reply]
Hi...

The module works fine now...
I have only problem with audio toner.. is different from original

I have to set somethings?
![20240703_214127.jpg](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/52530939/0d20ed40-2018-41d2-a635-d58794fc1ba0)

**Reply by visualapproach** [reply]
Nice! The audio is supposed to be different. Original beeps was boring 🥱

**Reply by hantrax** [reply]
Thanks!!
And compliments for your work!!!
